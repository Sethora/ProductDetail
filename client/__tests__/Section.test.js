import React from 'react';
import { shallow } from 'enzyme';
import Section from '../src/components/section';

describe('Section component', () => {
  test('renders the component', () => {
    const wrapper = shallow(<Section selected={{}} />);
    expect(wrapper).toMatchSnapshot();
  });
});