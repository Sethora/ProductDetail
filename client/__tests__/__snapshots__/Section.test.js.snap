// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section component renders the component 1`] = `
<styled.div>
  <Detail
    title=""
  />
</styled.div>
`;
